mlflow/store/artifact/databricks_artifact_repo.py

```diff
@@ -0,0 +1,152 @@
+            "path": path
+        }
+
+    def _get_write_credentials(self, run_id, path=None):
+        json_body = message_to_json(GetCredentialsForWrite(run_id=run_id, path=path))
+        return self._call_endpoint(DatabricksMlflowArtifactsService,
+                                   GetCredentialsForWrite, json_body)
+
+    def _get_read_credentials(self, run_id, path=None):
+        json_body = message_to_json(GetCredentialsForRead(run_id=run_id, path=path))
+        return self._call_endpoint(DatabricksMlflowArtifactsService,
+                                   GetCredentialsForRead, json_body)
+
+    def _azure_upload_file(self, credentials, local_file, artifact_path):
+        try:
+            if os.path.getsize(local_file) < _AZURE_SINGLE_BLOCK_BLOB_MAX_SIZE:
+                signed_write_uri = credentials.signed_uri
+                service = BlobClient.from_blob_url(blob_url=signed_write_uri, credential=None)
+                with open(local_file, "rb") as data:
+                    service.upload_blob(data, overwrite=True)
+            else:
+                uploading_block_list = list()
+                for chunk in yield_file_in_chunks(local_file, _AZURE_MAX_BLOCK_CHUNK_SIZE):
+                    signed_write_uri = self._get_write_credentials(self.run_id, artifact_path).credentials.signed_uri
+                    service = BlobClient.from_blob_url(blob_url=signed_write_uri, credential=None)
+                    block_id = base64.b64encode(uuid.uuid4().hex.encode())
+                    service.stage_block(block_id, chunk)
+                    uploading_block_list.append(block_id)
+                signed_write_uri = self._get_write_credentials(self.run_id, artifact_path).credentials.signed_uri
+                service = BlobClient.from_blob_url(blob_url=signed_write_uri, credential=None)
+                service.commit_block_list(uploading_block_list)
+        except Exception as err:
+            raise MlflowException(err)
+
+    def _azure_download_file(self, credentials, local_path):
+        signed_read_uri = credentials.signed_uri
+        service = BlobClient.from_blob_url(blob_url=signed_read_uri, credential=None)
+        try:
+            with open(local_path, "wb") as output_file:
+                blob = service.download_blob()
+                output_file.write(blob.readall())
+        except Exception as err:
+            raise MlflowException(err)
+
+    def _aws_upload_file(self, credentials, local_file):
+        pass
+
+    def _aws_download_file(self, credentials, local_path):
+        pass
+
+    def _upload_to_cloud(self, cloud_credentials, local_file, artifact_path):
+        if cloud_credentials.credentials.type == 1:
+            self._azure_upload_file(cloud_credentials.credentials, local_file, artifact_path)
+        else:
+            raise MlflowException('Not implemented yet')
+
+    def _download_from_cloud(self, cloud_credentials, local_path):
+        if cloud_credentials.credentials.type == 1:
```

Comment:
Nit: Can we do this instead for readability?:

```suggestion
        if cloud_credentials.credentials.type == ArtifactCredentials.AZURE_SAS_URI:
```

Reply:
Done